core/domain/acl_decorators.py

```diff
@@ -34,18 +34,42 @@
 
 
 def open_access(handler):
-    """Decorator to give access to everyone."""
+    """Decorator to give access to everyone.
+
+    Args:
+        handler: function. The function to be decorated.
```

Comment:
Please note that type info here should be what you get in python after executing `type(variable)`. So you might want to crosscheck the data type of `handler`. Ditto everywhere else in the file.

Reply:
Hi @vibhor98 -- actually, I think we had this discussion in an earlier PR, and eventually settled on "function" for cases like this. So I think "function" is OK?

Reply:
Ok, it's fine if we settled on it previously.